bundle/config/mutator/python/python_logger.go

```diff
@@ -0,0 +1,93 @@
+package python
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"strings"
+
+	"github.com/databricks/cli/libs/log"
+)
+
+type stderrLogger struct {
+	ctx context.Context
+	buf bytes.Buffer
+}
+
+type level string
+
+const levelError level = "ERROR"
+const levelInfo level = "INFO"
+const levelDebug level = "DEBUG"
+const levelWarning level = "WARNING"
+
+type logEntry struct {
+	Level   level  `json:"level,omitempty"`
+	Message string `json:"message"`
+}
+
+func NewStderrLogger(ctx context.Context) io.Writer {
```

Comment:
This feels more like a "PythonLogProxy" or something.

Reply:
And if it works with stdlib logging in Python, then it makes sense to keep it under e.g. `libs/python/logging`.

Reply:
I've removed most of it for now and called simply "logWriter"